### PR TITLE
mp-b2c-concrete-products-endpoint

### PR DESCRIPTION
**Pull request by himanshu-saxena-metacube**
mp-b2c-concrete-products-endpoint
https://spryker.atlassian.net/browse/CC-16228

### CODE REVIEW

File: tests/api/mp_b2c/glue/concrete_products_endpoints/concrete_products/positive.robot [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    And Response body parameter should be:    [data][type]   concrete-products
+    And Response body parameter should be:    [data][attributes][sku]    ${concrete_product_multiple_image_set}
+    And Response body parameter should be:    [data][attributes][name]    ${concrete_product_multiple_image_set_name}
+    And Response should contain the array of a certain size:    [data][attributes][attributeNames]    6
+    And Response should contain the array of a certain size:    [data][attributes][attributes]    6
+    And Response should contain the array of a certain size:    [data][relationships]   1
+    And Response include should contain certain entity type:    abstract-products
+    And Response include element has self link:   abstract-products
+
+Get_concrete_product_with_included_product_labels
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/concrete_products_endpoints/concrete_products/positive.robot:76]: test fails
image

Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2c/glue/concrete_products_endpoints/concrete_products/positive.robot:76]: Fixed, looks like someone has removed product label from back-end, hence updated the other one

File: tests/api/mp_b2c/glue/concrete_products_endpoints/concrete_products/positive.robot [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    And Response should contain the array of a certain size:    [data][attributes][attributes]    6
+    And Response body parameter should not be EMPTY:    [data][attributes][description]
+    And Response body parameter should not be EMPTY:    [data][attributes][productAbstractSku]
+    And Response body parameter should not be EMPTY:    [data][attributes][metaKeywords]
+    And Response body parameter should not be EMPTY:    [data][attributes][metaDescription]
+    And Response body parameter should be:    [data][attributes][averageRating]    None
+    And Response body parameter should be:    [data][attributes][reviewCount]    0
+    And Response body has correct self link internal
+
+Get_concrete_product_with_included_image_sets
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/concrete_products_endpoints/concrete_products/positive.robot:30]: it now fails
image

Comment (himanshu-saxena-metacube) [reply tests/api/mp_b2c/glue/concrete_products_endpoints/concrete_products/positive.robot:30]: Fixed

**Pull request merged**